openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -918,3 +920,118 @@ def test_on_jwt_headers_rejection(self):
             expected_status=status.HTTP_401_UNAUTHORIZED,
             expected_activation_status=True
         )
+
+
+@unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Account APIs are only supported in LMS')
+class TestAccountRetireMailings(TestCase):
```

Comment:
Is there any portion of this test class that can be factored out into common code for other LMS endpoints?

Reply:
I don't think a lot of this carries over elsewhere, but I will break out the permissions checking into a different set of tests since it really doesn't belong here and I don't want people working on the other endpoints thinking it's something we need to check in every set of endpoint tests.